### DISCUSSION TITLE
Non-interactive / batch functionality

### DISCUSSION THREAD

**Discussion by crosslightde** [d99kris/nchat#212] [Q&A]
Is there some sort of batch functionality? If not, are there plans on implementing something like that. For my home automation, I'd like to send message via WhatsApp (just a few per week to very few people). Info about fire alert, washing machine finished, etc. Would be awesome if I was able to do that via an actively developed and working product like nchat.

**Comment by d99kris** [comment]
Hi @crosslightde - no it's not supported, and unfortunately out of scope for this project.
